README.md

```diff
@@ -66,7 +66,24 @@ Now, to connect and use the database, go to Arcade, create a new data source wit
 * password: admin
 
 Save and start the index process of the data source.
- 
+
+## OrientDB 3.0.x: explore the demo database with Gremlin
```

Comment:
could you please fix the title of this section

Reply:
fixed !